gradle/project.gradle.kts

```diff
@@ -74,8 +77,8 @@ allprojects {
 dependencies {
 
     // Kotlin
-    compile("org.jetbrains.kotlin", "kotlin-stdlib-jdk8", kotlinVersion)
-    compile("org.jetbrains.kotlin", "kotlin-reflect", kotlinVersion)
+    compile(kotlin("stdlib"))
+    compile(kotlin("reflect"))
```

Comment:
this new code doesn't need to specify kotlinVersion and stuff. It's a convenience feature of kotlin-dsl - seems much more clear and concise to me, since these are essentially language libraries

Reply:
It brings in additional constructs developer needs to understand for no real benefit, but this lvl of trade-off is ok.

Reply:
needs to understand? compile-kotlin-stdlib - pretty obvious in my eyes. There is no relevant information in the old version that helps you understand it better

Reply:
Yes, developer no longer understands which version is being used and where do the libs come from (org.jetbrains.kotlin), because this information is not explicit. Sometimes less is not more. For all dev reading it knows - this can be a random library with names similar to kotlin libs.